Sources/MapboxNavigation/NavigationMapView+VanishingRouteLine.swift

```diff
@@ -323,6 +323,9 @@ extension NavigationMapView {
 
                 guard case let .lineString(lineString) = feature.geometry,
                       let distance = lineString.distance() else {
+                    if gradientStops.isEmpty {
```

Comment:
Is it's possible to simulate a case when `gradientStops` contain invalid value? If so, would be create to add unit-test to verify whether your new fix works in such case.

Reply:
Thanks for the suggestion, the test case is added. If could happen if the `CongestionSegment` is empty to generate a feature with empty coordinates `lineString`.

Reply:
One possibility would be a route that doesn’t contain this kind of annotation at all, such as a cycling route.